qiskit/circuit/library/evolved_operator_ansatz.py

```diff
@@ -30,6 +31,12 @@
 class EvolvedOperatorAnsatz(NLocal):
     """The evolved operator ansatz."""
 
+    @deprecate_func(
```

Comment:
Since both imports of `EvolvedOperatorAnsatz` from `qiskit.circuit.library` and `qiskit.circuit.library.n_local` should work, I'm not sure what should be the right message here.
@ElePT - could you please advise?

Reply:
I believe that this is one of the edge cases of our deprecation policy. We normally have to follow the: `create new path-deprecate old path-remove old path` sequence. But in this case, the new path is included in the old one. Given that the `EvolvedOperatorAnsatz` is still importable from `qiskit.circuit.library`, users won't really feel a change, so I think the best course of action in this case is not to raise any depreaction warning (even if it might sound a bit contradictory). I do think it would be nice to keep a release note notifying the change.

Reply:
I made the changes to switch from the deprecation to the upgrade/immediate replacement.